dev/TEAL.md

```diff
@@ -111,21 +111,21 @@ Some of these have immediate data in the byte or bytes after the opcode.
 | --- | --- |
 | `intcblock` | load block of uint64 constants |
 | `intc` | push value from uint64 constants to stack by index into constants |
```

Comment:
We've just realized that it may be confusing that the instruction is called `int` while it's actually only handling `uint64`.

Reply:
and if we ever add signed ints they'll be called `sint`.
If languages can have default-immutable "variables" then we can have default-unsigned integers. I think we're clear in dozens of places that the ints are unsigned.